src/fastjet/_generalevent.py

```diff
@@ -627,6 +627,35 @@ def constituents(self, min_pt):
         res = ak.Array(self._replace_multi())
         return res
 
+    def exclusive_jets_constituents(self, njets):
+        if njets <= 0:
+            raise ValueError("Njets cannot be <= 0")
+
+        self._out = []
+        self._input_flag = 0
+        for i in range(len(self._clusterable_level)):
+            np_results = self._results[i].to_numpy_exclusive_njet_with_constituents(
+                njets
+            )
+            off = np.insert(np_results[-1], 0, 0)
+            out = ak.Array(
+                ak.layout.ListOffsetArray64(
+                    ak.layout.Index64(np_results[0]),
+                    ak.layout.NumpyArray(np_results[1]),
+                ),
+                behavior=self.data.behavior,
+            )
+            outputs_to_inputs = ak.Array(
```

Comment:
Up until this point, the basic code of `exclusive_jets_constituents` and `exclusive_jets_constituent_index` is the same. I wonder if `exclusive_jets_constituents` should call `exclusive_jets_constituent_index` or will this cause unnecessary overhead? 

A similar situation applies to `constituent_index` and `constituents`. Clearly, in that case, it was chosen to do it this way. So, this approach is probably the right way to go.

If it turns out it is useful to minimize code duplication, we can deal with it in a follow-up PR.